scripts/atomic_commit_manager.py

```
@@ -0,0 +1,269 @@
+            print(f"Error executing commit group: {e}")
+            return False
+            
+    def _commit_to_worktree(self, worktree_path: Path, files: List[str], 
+                          commit_message: str) -> Optional[str]:
+        """Commit files to a specific worktree."""
+        try:
+            # Add files to git
+            for file_path in files:
+                full_path = worktree_path / file_path
+                if full_path.exists():
+                    subprocess.run(["git", "add", str(full_path)], 
```

Ruff (0.14.3)

`subprocess` call: check for execution of untrusted input

(S603)